### ISSUE TITLE
JSONDecodeError on large views (chunk is split on the middle)

### ISSUE THREAD

**Issue by shanipribadi** [aio-libs/aiocouchdb#8]
I'm using this code 
```
@asyncio.coroutine
def go():
    db = yield from couch_server.db('db)
    ddoc = yield from gobox_db.ddoc('ddoc')
    viewdoc = yield from ddoc.view('viewdoc')
    while True:
        res = yield from viewdoc.next()
        pprint.pprint(res['value'])
```
viewdoc.next() would return a JSONDecodeError exception on some rows.
by adding print(chunk) on feeds.py I found on some rows the chunk ends in the middle of the json object.

**Comment by kxepal** [comment]
Interesting. Do you have any proxy in the middle? Don't you eventually try to use aiocouchdb against 2.0 or cloudant?

**Comment by shanipribadi** [comment]
no proxy, I tried against a local server. the db has about 4000 documents, each rows is about 1200 bytes long.. catching the exception shows that it occurs multiple times.

There's this message printed at the end.
```
<CIMultiDictProxy('TRANSFER-ENCODING': 'chunked', 'SERVER': 'CouchDB/1.6.1 (Erlang OTP/18)', 'ETAG': '"4S06NFX83NJPT9LZ7E867LV0M"', 'DATE': 'Tue, 17 Nov 2015 12:46:06 GMT', 'CONTENT-TYPE': 'application/json', 'CACHE-CONTROL': 'must-revalidate')>
```

Any idea how to debug it?

**Comment by kxepal** [comment]
Hm, interesting and strange. However, I know the solution. This still have to be fixed since 2.0 stops sending complete objects with a single chunk by default which breaks this logic which seems is broken already for unknown reason.

**Comment by kxepal** [comment]
Sorry for a long delay! This should be fixed by 6e63d05. If it still not, please reopen this issue.
